app/src/main/java/edu/agh/susgame/front/ui/component/menu/CreateGameView.kt

```diff
@@ -1,32 +1,182 @@
 package edu.agh.susgame.front.ui.component.menu
 
+import android.widget.Toast
+import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Visibility
+import androidx.compose.material.icons.filled.VisibilityOff
 import androidx.compose.material3.Button
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Slider
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.PasswordVisualTransformation
+import androidx.compose.ui.text.input.VisualTransformation
+
 import androidx.navigation.NavController
 import edu.agh.susgame.front.ui.Translation
 import edu.agh.susgame.front.ui.component.common.Header
 import edu.agh.susgame.front.ui.component.menu.navigation.MenuRoute
 import edu.agh.susgame.front.ui.theme.PaddingL
 
+import java.util.Locale
+import kotlin.math.roundToInt
+
+
+private const val maxPinLength = 6
+private const val maxPlayersAmount = 6
+private const val minPlayersAmount = 2
+private const val defaultPlayersAmount = 4
+private const val minGameTime = 5f
+private const val maxGameTime = 15f
+private const val defaultGameTime = 10
 @Composable
 fun CreateGameView(
     navController: NavController,
 ) {
-    Column(Modifier.padding(PaddingL)) {
+    var gameName by remember { mutableStateOf(Translation.CreateGame.DEFAULT_GAME_NAME) }
```

Comment:
Moim zdaniem spoko byłoby, żeby domyślnie nazwa gry była pusta, ale tego typu wartość fajnie dać jako placeholder (czyli takie na szaro, co znika gdy się coś wpisze)

Reply:
własnie tutaj myslalem nad tym, aby ta domyslna nazwa była dynamicznie tworzona za pomocą jego nicku i jakichs losowych cyfr, dla gracza zapewne nie ma znaczenia nazwa gry, służy ona tylko do tego, aby inni ją odnaleźli, więc zaproponowanie domyślnej ułatwiałoby korzystanie z tego interfejsu, ale myślę ze na razie to az tak wazne nie jest, narazie ustawie DEFAULT_GAME_NAME na pusty string

Reply:
Aaaa okej, ma dużo sensu - bardzo dobry pomysł. Zupełnie nie ogarnąłem że coś takiego miałeś w głowie, myślałem że nazwa ma być ustawiona na sztywno

Jedyny problem, to nie wiem czy nie ustaliliśmy koniec końców, że nick będzie wpisywany od zera za każdym razem przy dołączaniu/tworzeniu gry, ale to do ogarnięcia w przyszłości, na razie jest spoko. Możesz nawet przywrócić na jakieś "Gra uqsz'a"